### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by tim77**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,1271 @@
...
+msgid "Sorry, command parse failed, please report a bug to <"
+msgstr "Обработка команды не выполнена, сообщите об ошибке <"
+
+#: ../data/ui/app.glade:115 ../data/ui/app.glade:116
+msgid "Srain"
+msgstr "Srain"
+
+#: ../data/ui/prefs_dialog.glade:9
+msgid "Srain Perferences"
+msgstr "Параметры Strain"
```
Comment (SilverRainZ) [inline po/ru.po]: Typo.

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,1271 @@
...
+msgid "Received unsupported CTCP %1$s response form %2$s"
+msgstr ""
+
+#: ../src/core/chat_command.c:446 ../src/core/app_ui_event.c:156
+msgid "Reconnection stopped"
+msgstr "Переподключение остановлено"
+
+#: ../data/ui/prefs_dialog.glade:107
+msgid "Reload Configuration"
+msgstr ""
```
Comment (SilverRainZ) [inline po/ru.po]: Can this string to be translated?

Comment (SilverRainZ) [reply po/ru.po]: It is a tilte of preference dialog tab page.

File: po/ru.po [hunk 3]
```diff
@@ -0,0 +1,1271 @@
...
+msgid "Please connect to server before joining any channel"
+msgstr "Подключитесь к серверу, прежде чем присоединяться к каналу"
+
+#: ../src/sui/sui_join_panel.c:377
+msgid "Please connect to server before searching any channel"
+msgstr "Подключитесь к серверу перед поиском канала"
+
+#: ../data/ui/connect_panel.glade:106
+msgid "Quick Mode"
+msgstr ""
```
Comment (SilverRainZ) [inline po/ru.po]: Can this string to be translated?
If you don‘t know the context of this text, free feel to ask me.

This text is title of tab page of server connection panel, this page provides a easy(and quick) way to connect to IRC server.

File: po/ru.po [hunk 4]
```diff
@@ -0,0 +1,1271 @@
...
+
+#: ../src/core/app_irc_event.c:927
+#, c-format
+msgid "Received CTCP %1$s request form %2$s"
+msgstr ""
```
Comment (SilverRainZ) [inline po/ru.po]: CTCP is a special IRC message, see https://en.wikipedia.org/wiki/Client-to-client_protocol
%1$s is CTCP message type and %2$s is message sender.

**Comment by tim77** [comment]
Sorry, i messed a little bit with git. 🤦‍♂️ Fixed in new commit.

File: po/ru.po (hunk 4 above)
Comment (tim77) [reply po/ru.po]: BTW, this string
```
msgid "Received CTCP %1$s request **form** %2$s" 
```
could be there is a type in **form** word?

Comment (SilverRainZ) [reply po/ru.po]: Yes, I will fix it later.

**Comment by SilverRainZ** [comment]
Can I merge it now?

**Comment by tim77** [comment]
I think yes. I followed Gnome translation guidelines but some things i still think how to translate better. IMO this could be done after few days of using App.

**Comment by SilverRainZ** [comment]
The i18n function is broken due to the change of Makafile, #176 fixes this problem, I will merge it ASAP.

**Pull request merged**